src/Core/Domain/Product/Query/GetPackedProducts.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\Query;
+
+use PrestaShop\PrestaShop\Core\Domain\Language\ValueObject\LanguageId;
+use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
+
+/**
+ * Retrieves product from a pack
+ */
+class GetPackedProducts
+{
+    /**
+     * @var ProductId
+     */
+    private $packId;
+
+    /**
+     * @var LanguageId
+     */
+    private $languageId;
+
+    /**
+     * @param int $packId id of product which represents the pack
+     * @param int $languageId
+     */
+    public function __construct(int $packId, int $languageId)
+    {
+        $this->packId = new ProductId($packId);
+        $this->languageId = new LanguageId($languageId);
+    }
+
+    /**
+     * @return ProductId
+     */
+    public function getPackId(): ProductId
+    {
+        return $this->packId;
+    }
+
+    /**
+     * @return LanguageId
+     */
+    public function getLanguageId(): LanguageId
```

Comment:
I'm not sure this is really useful to set the language id, especially since `PackedProduct` doesn't contain any localized value You could simply use the default language id for `Pack:getItems`
Or maybe it would be useful to have more info about the product from the pack, like their name and images Do you think it's something that will be added later?